pytorch_lightning/utilities/apply_func.py

```diff
@@ -86,9 +90,11 @@ def move_data_to_device(batch: Any, device: torch.device):
         - :meth:`torch.Tensor.to`
         - :class:`torch.device`
     """
-
     def batch_to(data):
-        if isinstance(data, Batch):
+        if Batch is None:
+            raise ImportError('You want to use `torchtext` package which is not installed yet,'
+                              ' install it with `pip install torchtext`.')
```

Comment:
this is incorrect, it will show the message to users that don't use torchtext. we need to wrap the logic here with `TORCHTEXT AVAILABLE`.